src/s1ap/main.c

```diff
@@ -248,12 +248,22 @@ accept_sctp(void *data)
 
 			if (FD_ISSET(sd, &readfds)) {
 				if ((valread = recv_sctp_msg(sd, buffer, SCTP_BUF_SIZE)) <= 0) {
-
                     struct EnbStruct temp = {0};
+                    int temp_cbIndex = findControlBlockWithEnbFd(sd);
+                    s1_incoming_msg_data_t s1Msg={0};
+                    s1Msg.msg_type = enb_status_msg; 
+                    s1Msg.srcInstAddr = htonl(s1apAppInstanceNum_c);
+                    s1Msg.destInstAddr = htonl(mmeAppInstanceNum_c);
+                    struct s1apEnbStatus_Msg *enbMsg = &s1Msg.msg_data.s1ap_enb_status_m; 
+                    enbMsg->ver = 1;
+                    enbMsg->status = 0;
+                    enbMsg->context_id = temp_cbIndex;
+                    send_tipc_message(ipc_S1ap_Hndl, mmeAppInstanceNum_c, (char *)&s1Msg, S1_READ_MSG_BUF_SIZE);
+                    
                     clearControlBlockDetailsEnbFd(sd, &temp);
```

Comment:
since we do clear enb context here.. we won't get the restart counter updated I think.. 
we could probably add a timestamp here while sending stats ??

Reply:
timestamp comes prometheus. We will not have restart_counter correct always.